### PR TITLE
docs: rework print debug logs

### PR DESCRIPTION
**Pull request by MikeMcC399**
## Issues

In [References > Troubleshooting > Print DEBUG logs](https://docs.cypress.io/guides/references/troubleshooting#Print-DEBUG-logs)

- Instructions do not work when copied and pasted into a terminal window. They are missing a package manager prefix (`npx`, `yarn` or `pnpm`).
- Instructions for Windows PowerShell are missing.

## Change

- Show explicit instructions for debugging with npm, Yarn and pnpm using tabbed display.

- Show instructions for setting `DEBUG` with Windows PowerShell.

- Harmonize the use of prefixes `npx` / `yarn` / `pnpm` in command lines.

- Correct non-working debug module names using working module names.

### CODE REVIEW

**Comment by MikeMcC399** [comment]
Preview

---
**BEFORE**

Print-debug-logs-before

---
**AFTER**

https://github.com/MikeMcC399/cypress-documentation/blob/rework/print-debug-logs/docs/guides/references/troubleshooting.mdx#print-debug-logs

Print-debug-logs-after

File: docs/guides/references/troubleshooting.mdx [hunk 1]
```diff
@@ -247,34 +247,69 @@ an issue you have by clearing this app data.
 
 ## Print DEBUG logs
 
-Cypress is built using the [debug](https://github.com/visionmedia/debug) module.
-That means you can receive helpful debugging output by running Cypress with this
-turned on. **Note:** you will see a LOT of messages when running with
+Cypress is built using the [debug](https://github.com/debug-js/debug) module.
```
Comment (jennifer-shehane) [inline docs/guides/references/troubleshooting.mdx:250]: I know you didn't write this, but this is a really weird statement. 

```suggestion
Cypress utilizes the [debug](https://github.com/debug-js/debug) module.
```

File: docs/guides/references/troubleshooting.mdx [hunk 2]
```diff
@@ -247,34 +247,69 @@ an issue you have by clearing this app data.
...
+That means you can receive helpful debugging output by running Cypress with
+the environment variable `DEBUG` set to `cypress:*` prior to running `cypress run` or `cypress open`.
```
Comment (jennifer-shehane) [inline docs/guides/references/troubleshooting.mdx:252]: People have gotten confused with system env vars and Cypress env vars in the past.

```suggestion
That means you can receive helpful debugging output by running Cypress with a system-level environment variable `DEBUG` set to `cypress:*` prior to running `cypress run` or `cypress open`.
```

File: docs/guides/references/troubleshooting.mdx [hunk 3]
```diff
@@ -247,34 +247,69 @@ an issue you have by clearing this app data.
...
+**Note:** you will see a LOT of messages when running with the
```
Comment (jennifer-shehane) [inline docs/guides/references/troubleshooting.mdx:253]: Just noting some facts if you want to incorporate it. It's also possible that printing a large amount of DEBUG statement could impact performance of test runs, so it should really only be used when something specifically needs debugging. It's always best to try to target the least amount of logs identify to see the problem.

File: docs/guides/references/troubleshooting.mdx [hunk 4]
```diff
@@ -247,34 +247,69 @@ an issue you have by clearing this app data.
...
 `DEBUG=...` setting.
 
-**On Mac or Linux:**
+**On macOS, Linux or Windows (Git Bash):**
```
Comment (jennifer-shehane) [inline docs/guides/references/troubleshooting.mdx:256]: I would say that capturing logs in docker or CI is sometimes a challenge for people - not sure if you want to link to the other docs on that in docker-images / github actions, etc.

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]

File: docs/guides/references/troubleshooting.mdx (hunk 4 above)
Comment (MikeMcC399) [reply docs/guides/references/troubleshooting.mdx:255]: Good suggestion. I'll make a note to tackle this subject in a follow-on PR.

File: docs/guides/references/troubleshooting.mdx (hunk 3 above)
Comment (MikeMcC399) [reply docs/guides/references/troubleshooting.mdx]: Addressed in https://github.com/cypress-io/cypress-documentation/pull/5921/commits/d62fb2a091a634f9463813abf78913f8537e767f

**Comment by MikeMcC399** [comment]
Netlify previews are broken. Here is a local screenshot:

image

**Pull request merged**